README.md

```diff
@@ -14,6 +14,15 @@ If the above command fails or if it reports a version number significantly diffe
 
 ### Project setup
 
+Clone this repo:
+```
+git clone https://github.com/monarch-initiative/monarch-ui
+```
+Go into the directory that has your newly cloned copy of this repo:
+```
+cd monarch-ui
+```
+Then you can run the installation setup:
```

Comment:
Should we add a note here about node and npm versions and potentially link to [installing-nodejs](#installing-nodejs)?

Reply:
sure, do you want to add something? i did the npm setup a while ago and can't remember what exactly i did.

Reply:
These instructions are an improvement, but not quite correct. We want to tell people to FORK that repo, and then to CLONE their fork. This has been excruciating detailed in Legacy here: https://github.com/monarch-initiative/monarch-app/blob/master/CONTRIBUTING.md and much of that document should be repurposed for monarch-ui (I honestly thought I had done this a while ago).